bb_wrapper/wrapper/request.py

```diff
@@ -1,13 +1,35 @@
+from time import sleep
 from json.decoder import JSONDecodeError
 
 import requests
 
-from ..utils import _get_logger
+from requests import ConnectionError
+from urllib3.exceptions import ProtocolError
+
+from bb_wrapper.utils import _get_logger
 
 
 logger = _get_logger("requests")
 
 
+def retry_request(max_retries=5):
+    def wrapper(func):
+        def inner(*args, counter=None, **kwargs):
+            counter = counter if counter is not None else max_retries
+            try:
+                sleep(0.01)
```

Comment:
pode aumentar esse sleep.

Poderia até colocar um random de 0.01 a 0.9 nesse valor

```suggestion
                sleep_time = random(1, 90)/100
                sleep(sleep_time)
```